roles/qe-tendrl-repo/README.rst

```diff
@@ -0,0 +1,19 @@
+================
+ QE-Tendrl-repo
+================
+
+This role enables required Tendrl repositories and installs their
+public gpg rpm keys (if configured).
+
+By default `tendrl/release copr`_ and `tendrl/dependencies copr`_ repositories.
+
+The default repositories might be overwritten by following variables:
+  - tendrl_repo_url
+  - tendrl_repo_gpgkey_url
+  - tendrl_dependencies_repo_url
+  - tendrl_dependencies_repo_gpgkey_url
+
+If gpgkey url set to empty string (""), gpgcheck is disabled.
```

Comment:
I was thinking about it, but I don't think so, as it is defined in _defaults_. Or is there possibility, that  variable configured in _defaults_ will be undefined?